windows_containers/windows-containers-release-notes-7-x.adoc

```diff
@@ -45,6 +45,16 @@ This release of the WMCO provides new features and bug fixes for running Windows
 // Also in 10.15.0
 * Previously, because of bad logic in the networking configuration script, the WICD was incorrectly reading carriage returns in the CNI configuration file as changes, and identified the file as modified. This caused the CNI configuration to be unnecessarily reloaded, potentially resulting in container restarts and brief network outages. With this fix, the WICD now reloads the CNI configuration only when the CNI configuration is actually modified. (link:https://issues.redhat.com/browse/OCPBUGS-27771[*OCPBUGS-27771*])
 
+* Previously, the WMCO incorrectly approved the node certificate signing requests (CSR) for all nodes trying to join a cluster, not just Windows node CSRs. With this fix, the WMCO approves CSRs for only Windows nodes as expected. (link:https://issues.redhat.com/browse/OCPBUGS-27139[*OCPBUGS-27139*])
+
+// Also in 10.15.0
+* Previously, because of routing issues present in Windows Server 2019, under certain conditions and after more than one hour of running time, workloads on Windows Server 2019 could have experienced packet loss when communicating with other containers in the cluster. This fix enables Direct Server Return (DSR) routing within kube-proxy. As a result, DSR now causes request and response traffic to use a different network path, circumventing the bug within Windows Server 2019. (link:https://issues.redhat.com/browse/OCPBUGS-28254[*OCPBUGS-28254*])
+
+* Previously, because the upgrade path from WMCO 6.x to WMCO 7.x included previously released versions, the WMCO would fail during the upgrade. With this fix, you can successfully upgrade from WMCO 6.x to WMCO 7.x. (link:https://issues.redhat.com/browse/OCPBUGS-27775[*OCPBUGS-27775*])
+
+// Also in 10.15.0
+* Previously, because of a lack of synchronization between Windows machine set nodes and Bring-Your-Own-Host (BYOH) instances, during an update the machine set nodes and the BYOH instances could update simultaneously, which could have impacted running workloads. This fix introduces a locking mechanism so that machine set nodes and BYOH instances update individually. (link:https://issues.redhat.com/browse/OCPBUGS-23020[*OCPBUGS-23020*])
```

Comment:
WMCO is just compute machine sets, right? (I could def be misremembering, if so disregard)

```suggestion
* Previously, because of a lack of synchronization between Windows compute machine set nodes and Bring-Your-Own-Host (BYOH) instances, during an update the compute machine set nodes and the BYOH instances could update simultaneously, which could impact running workloads. This fix introduces a locking mechanism so that compute machine set nodes and BYOH instances update individually. (link:https://issues.redhat.com/browse/OCPBUGS-23020[*OCPBUGS-23020*])
```